src/webhooks/helius/helius.service.ts

```diff
@@ -102,10 +118,32 @@ export class HeliusService {
           itemsMinted: { increment: 1 },
         },
       });
+    const nftTransactionInfo = enrichedTransaction.events.nft as NFTEvent & {
+      amount: number;
+    };
+    const logIntoMintReceipt = this.prisma.mintReceipt.create({
+      data: {
+        buyer: nftTransactionInfo.buyer,
+        price: nftTransactionInfo.amount / LAMPORTS_PER_SOL, // not sure if there is an existing helper function that can convert to SOL properly (max 9 decimals I assume?)
```

Comment:
Let's remove this comment and store the Lamport value in the database so we don't have to use decimals/floats?